Dockerfile

```
@@ -1,15 +1,12 @@
 # syntax=docker/dockerfile:1
 
-FROM eclipse-temurin:11
-WORKDIR /app
+FROM amazoncorretto:11-alpine3.18
 
-COPY .mvn/ .mvn
-COPY mvnw pom.xml ./
+ENV APP_DIR=/app
 
-COPY checkstyle.xml .
+RUN mkdir $APP_DIR
+WORKDIR $APP_DIR
 
-RUN ./mvnw dependency:go-offline
+COPY build/libs/interview-planning-*.jar $APP_DIR/
```

```suggestion
COPY build/libs/interview-planning-*.jar $APP_DIR/interview-planning.jar
```

Your app version could be incremented. So, your docker file should not be updated all the time you update the app version.

also, update your `CMD`